dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -292,6 +292,37 @@ def profile(self):
 
         return profile
 
+    def report(self, remove_disabled_flag):
```

Comment:
should by default be `remove_disabled_flag=False`

Reply:
done